media-video/ffmpeg/ffmpeg-4.4.recipe

```diff
@@ -4,15 +4,15 @@ and stream audio and video. It includes libavcodec - the leading audio/video \
 codec library."
 HOMEPAGE="https://ffmpeg.org/"
 COPYRIGHT="2000-2003 Fabrice Bellard
-	2003-2020 the FFmpeg developers"
+	2003-2021 the FFmpeg developers"
 LICENSE="GNU LGPL v3"
-REVISION="2"
+REVISION="1"
 SOURCE_URI="https://ffmpeg.org/releases/ffmpeg-$portVersion.tar.xz"
-CHECKSUM_SHA256="1d0ad06484f44bcb97eba5e93c40bcb893890f9f64aeb43e46cd9bb4cbd6795d"
+CHECKSUM_SHA256="06b10a183ce5371f915c6bb15b7b1fffbe046e8275099c96affc29e17645d909"
 PATCHES="ffmpeg-$portVersion.patchset"
 ADDITIONAL_FILES="gcc_runtime.c"
 
-ARCHITECTURES="!x86_gcc2 x86 !x86_64"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86_gcc2 !x86"
```

Comment:
Is this correct? primary x86_gcc2 can't be build but for secondary it can, x86 can't be build for secondary architecture?

Reply:
It was so in the previous ffmpeg recipe, i just reenabled for 64bit.
Will clean it up.

Reply:
see https://dev.haiku-os.org/ticket/16294#comment:10

Reply:
@korli : sadly i can't seem to find any video hosting website which works with Web+.
Otter works ok with youtube and with ffmpeg 4.4